### ISSUE TITLE
On first load, editions don't display

### ISSUE THREAD

**Issue by kphillpotts** [weeklyxamarin/WeeklyXamarin.mobile#36]
Requires a refresh of the first screen

**Comment by saamerm** [comment]
![image](https://user-images.githubusercontent.com/8262287/120872028-6749e280-c56b-11eb-9778-e32f656c6476.png)

Just downloaded the app on my iPhone 8, I faced the same issue 
(Thought I'd add repro steps and a screenshot)

**Comment by kphillpotts** [comment]
Thanks @saamerm - but why? ;-)
I might need to look at device logs because I don't think it happens on simulators.

**Comment by kphillpotts** [comment]
@saamerm mentioned during James' stream that it happens when setting Link All.  so probably linker goodness.

**Comment by adenearnshaw** [comment]
When LinkAll is set, if I put a breakpoint inside EditionsViewModel.LoadEditionsCommand(), we do not hit it on first launch, but does if you then pull-to-refresh

**Comment by saamerm** [comment]
Exactly what @adenearnshaw said. There's no function that seems to call the List generation API on app load. So I basically called that function in the constructor.

**Comment by adenearnshaw** [comment]
@saamerm I think the solution may be more subtle than that.
From the XF source, the Command should be executed when the value of RefreshView.IsRefreshing is changed (https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core/RefreshView.cs)

The EditionsPage code behind, alters this value via the IsBusy property of the ViewModel. 

Given that it works when the Linker is off & in Android, I think there's something a little deeper going on. Just adding an additional call to fetch the data could cause it to run twice on Android which isn't ideal.

I've tried adding the following to the linker.xml, but that doesn't appear to have worked:
```xml
<assembly fullname="Xamarin.Forms.Platform.iOS">
    <type fullname="Xamarin.Forms.Platform.iOS.RefreshViewRenderer" />
</assembly>
```

**Comment by adenearnshaw** [comment]
I've done a bit more digging and found that rather than use the IsBusy property on BaseViewModel, if you replace it with a new INPC property in the ViewModel, then everything works as expected.

Given this information, I suspected the linker is stripping something out of MvvmHelpers.

By preserving that in the Linker.xml, using the code here, I can also get the app to function correctly:
```xml
<assembly fullname="MvvmHelpers">
  <type fullname="*" />
</assembly>
```

@kphillpotts do have a preference as to whether we replace the property, or preserve MvvmHelpers from the linker?

**Comment by kphillpotts** [comment]
For the sake of simplicity I think preserve MvvmHelpers would be fine.  It's not a big library anyway.

**Comment by adenearnshaw** [comment]
@saamerm given you've already got an open PR for this issue, I'm more than happy if you wanted to put the linker fix into that rather than me creating another fork/branch/pr?

**Comment by saamerm** [comment]
@adenearnshaw nah, I'll close mine. Good catch. 
It's strange that it doesn't work only once. I wonder what else is the linker & bug effecting

**Comment by adenearnshaw** [comment]
Forgot to mention that when I set the Droid app to Release, I was also able to replicate this issue so added the MvvmHelpers exclusion to the Android Linker file too.

Given this doesn't happen in the Play Store build, is it possible, that AppCenter is building in Debug mode & therefore not Linking anything?

**Comment by kphillpotts** [comment]
Fixed with pull request from #37 - Nice!
